Resources/Prototypes/_LostParadise/Flavors/flavors.yml

```
@@ -26,4 +26,14 @@
   id: flowertea
   flavorType: Complex
-  description: flavor-complex-flower-tea
\ No newline at end of file
+  description: flavor-complex-flower-tea
+
+- type: flavor
+  id: lppcargogargleblaster
+  flavorType: Complex
+  description: flavor-complex-lpp-cargo-gargle-blaster
+
+#- type: flavor
```

yamllint

[warning] missing starting space in comment

(comments)